src/libData/AccountData/AccountStore.h

```diff
@@ -67,16 +67,14 @@ class AccountStoreTemp : public AccountStoreSC<std::map<Address, Account>> {
 };
 
 // Singleton class for providing interface related Account System
-class AccountStore
-    : public AccountStoreTrie<std::unordered_map<Address, Account>>,
-      Singleton<AccountStore> {
+class AccountStore : public AccountStoreTrie, Singleton<AccountStore> {
   /// instantiate of AccountStoreTemp, which is serving for the StateDelta
   /// generation
   std::unique_ptr<AccountStoreTemp> m_accountStoreTemp;
 
   /// used for states reverting
-  std::unordered_map<Address, Account> m_addressToAccountRevChanged;
-  std::unordered_map<Address, Account> m_addressToAccountRevCreated;
+  std::map<Address, Account> m_addressToAccountRevChanged;
```

Comment:
Any reason to change from unordered to ordered? Because unordered map actually gives faster access

Reply:
Upon testing, nodes could not reach consensus if I used unordered, so I presumed it was due to account ordering in AccountStoreTemp. Not sure if we iterate across AccountStoreTemp during consensus (is state delta ordering part of consensus at any stage?). I will look into it further after the next release to see where exactly the ordering matters and if we can somehow just switch to unordered map everywhere to preserve performance.